frontend/types/mapbox__react-click-to-select/index.d.ts

```diff
@@ -0,0 +1,11 @@
+declare module '@mapbox/react-click-to-select' {
```

Comment:
does the double underscore in the file name have any particular meaning (either in TypeScript and/or React)?

Reply:
Yes, the double underscore is the convention for a file with type defs for a scoped package. If it were named differently, TypeScript would not find the types.